bin/i18n/sync-out.rb

```diff
@@ -200,7 +198,7 @@ def distribute_translations
   total_locales = Languages.get_locale.count
   Languages.get_locale.each_with_index do |prop, i|
     locale = prop[:locale_s]
-    print "#{CLEAR}Distributing #{locale} (#{i}/#{total_locales})"
+    print "Distributing #{locale} (#{i}/#{total_locales})"
```

Comment:
(mentioned this offline, repeating here for posterity)

these prints should be updated to puts now that we aren't CLEARing manually